core/server/lib/fs/read-csv.js

```diff
@@ -26,11 +26,6 @@ module.exports = function readCSV(options) {
                         result[columnsToExtract[0].name] = value[headers[0]];
                         return result;
                     });
-
-                    // Add first row
```

Comment:
This change works but the comment above: `// If CSV is single column - return all values including header` made me think twice about why this behavior is implemented in such way. If we are removing this bit, it would actually make sense to remove a whole wrapping `if` starting at [:23](https://github.com/TryGhost/Ghost/pull/10229/files#diff-605cc245eadc8824fe4dd429b3b90108L23). @lunaticmonk could you please expand the `subscribers_spec.js` for both api's using `two-columns-with-header.csv` as well to see if this module could be simplified further :)
@kirrg001 do you have any insight into why csv is treating single-column csv differently? haven't found any good usecases in the codebase.

Reply:
@gargol This looks like a pure logical bug to me.

Regarding the refactoring: Subscribers is a beta feature and we are currently working on the members feature. I don't see any need here to refactor this code. 

My suggestion would be: merge 😆

Reply:
@gargol I have checked both the api's using ```two-columns-with-header.csv``` as well before creating the pull request. Both of them passed.

Reply:
Ok. Merge it is then :smile: